rootfs/Dockerfile

```diff
@@ -1,41 +1,24 @@
-FROM quay.io/deis/base:v0.3.5
+FROM registry:2.5.1
 
-COPY . /
+RUN apk add --no-cache \
+        gcc \
+        git \
+        linux-headers \
```

Comment:
Yeah we can improve upon this to add and purge gcc, linux-headers etc. It would just increase build time significantly on PRs, but it'd be worth exploring.